tensorflow/lite/micro/examples/micro_speech/README.md

```diff
@@ -214,28 +214,36 @@ The next steps assume that the
 
 ### Generate the examples
 
-The example project can be generated with the following command: `make -f
-tensorflow/lite/micro/tools/make/Makefile TARGET=esp
-generate_micro_speech_esp_project`
+The example project can be generated with the following command:
+```
+make -f tensorflow/lite/micro/tools/make/Makefile TARGET=esp generate_micro_speech_esp_project
+```
```

Comment:
this formatting will likely get un-done by the internal CI system.

Adding this line to the top of the file should help:
```md
<!-- mdformat off(b/169948621#comment2) -->
```

similar to https://github.com/tensorflow/tensorflow/commit/c702a443f55171b8d3bddc4a02980d6ad3fc9fea

Reply:
Thanks, that explains line breaks in READMEs!